src/server/pathway-commons/search/index.js

```diff
@@ -1,7 +1,7 @@
 const {search, utilities} = require('pathway-commons');
 const path = require('path');
 const _ = require('lodash');
-const getHGNCData = require('./hgnc');
+const geneValidator = require('../../enrichment-map/gene-validator/index').validatorGconvert;
```

Comment:
Doesn't need to be changed but just FYI: You don't need to specify `index` or `index.js`.  Specifying the dir implicitly refs the index file